src/pages/EventAdministration/components/EventFormAdmin.tsx

```diff
@@ -25,21 +127,72 @@ const EventFormAdmin = ({ eventId }: EventFormAdminProps) => {
 
   const EventFormEditor = ({ formType }: EventFormEditorProps) => {
     const createForm = useCreateForm();
+    const showSnackbar = useSnackbar();
 
-    const newForm: EventFormCreate = {
+    const { data: formtemplates = [] } = useFormTemplates();
+    const [isCreatingForm, setIsCreatingTemplate] = useState(false);
+    const [formtemplateTitle, setFormtemplateTitle] = useState('');
+    const [formtemplateId, setFormtemplateId] = useState('-');
+    const { data: form, isLoading, isError } = useFormById(formtemplateId);
+
+    const newForm: FormCreate = {
       title: `${event.title} - ${formType === EventFormType.SURVEY ? 'påmeldingsskjema' : 'evalueringsskjema'}`,
       type: formType,
       event: event.id,
       resource_type: FormResourceType.EVENT_FORM,
       fields: [],
     };
 
+    const newFormTemplateCreate: FormCreate = {
+      title: 'EMPTY',
+      resource_type: FormResourceType.FORM,
+      fields: [],
+      viewer_has_answered: false,
+      template: true,
+    };
+
     const onCreate = async () => createForm.mutate(newForm);
 
+    const onCreateTemplate = async () => {
+      createForm.mutate(newFormTemplateCreate, {
+        onSuccess: (data) => {
+          showSnackbar(data.title, 'success');
+          setFormtemplateId(data.id);
+        },
+        onError: (e) => {
+          showSnackbar(e.detail, 'error');
+        },
+      });
+      setIsCreatingTemplate(true);
+    };
+
     return (
-      <Button fullWidth onClick={onCreate} variant='outlined'>
-        Opprett {formType === EventFormType.SURVEY ? 'påmeldingsskjema' : 'evalueringsskjema'}
-      </Button>
+      <>
+        <Button fullWidth onClick={onCreate} variant='outlined'>
+          Opprett {formType === EventFormType.SURVEY ? 'påmeldingsskjema' : 'evalueringsskjema'}
+        </Button>
+        <Expand flat header='Bruk en mal' sx={{ mt: 1 }}>
+          {isCreatingForm ? (
+            <>
+              <TextField label='Navn' onChange={(e) => setFormtemplateTitle(e.target.value)} sx={{ mb: 1 }} type='text' value={formtemplateTitle} />
+              {!isLoading && !isError && form ? <FormFieldsEditor form={form} newTitle={formtemplateTitle}></FormFieldsEditor> : ''}
```

Comment:
Her kunne du vel bruke onSave?
```suggestion
              {!isLoading && !isError && form && <FormFieldsEditor onSave={() => setIsCreatingTemplate(false)} form={form} newTitle={formtemplateTitle}></FormFieldsEditor>}
```